etl/steps/data/grapher/who/2024-06-17/guinea_worm.meta.override.yml

```diff
@@ -0,0 +1,31 @@
+tables:
+  guinea_worm:
+    variables:
+      time_frame_certified:
+        description_key: []
+        description_processing: |-
+          The current and historical values for the status of Guinea worm disease (Dracunculiasis) as certified by the WHO. To be certified as free of guinea worm disease, a country must have reported zero indigenous cases through active surveillance for at least three consecutive years.
+          Data regarding certification status is taken from: https://web.archive.org/web/20211024081702/https://apps.who.int/dracunculiasis/dradata/html/report_Countries_t0.html
+
+          This is supplemented with more recent changes to Guinea worm disease certification:
+          - Angola has had endemic status since 2020:  https://www.who.int/news/item/23-09-2020-eradicating-dracunculiasis-human-cases-and-animal-infections-decline-as-angola-becomes-endemic
```

Comment:
You could be a little bit more explicit that we have actively added this information to the dataset (some discussion of the type of voice we should use in `description_processing` [here.](https://owid.slack.com/archives/C03NA1B2P1U/p1709129035048969)